gis4wrf/plugin/ui/widget_geo.py

```diff
@@ -61,8 +61,9 @@ def populate_tree(self) -> None:
             item.setFlags(item.flags() | Qt.ItemIsUserCheckable)
             if is_geo_dataset_downloaded(id, self.options.geog_dir):
                 item.setFlags(Qt.NoItemFlags)
-                item.setToolTip(0, 'Dataset downloaded in: {}'.format(
-                    get_geo_dataset_path(id, self.options.geog_dir)))
+                for col in range(3):
```

Comment:
Only the first column should have this tooltip. The description column should have the description tooltip since that is often too long.

Reply:
Sure, I see what you mean!